templates/addons/linode-cloud-controller-manager-helm.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
```

Comment:
considering this is needed for base cluster functionality, should we just include this in the cluster-template.yaml(along with the cilium config?)

Reply:
I was thinking about that earlier, sounds good.